app/version.properties

```diff
@@ -0,0 +1,6 @@
+#
+#This file was automatically generated by 'versioning-plugin'. DO NOT EDIT MANUALLY.
+#
+#Sun Jan 17 12:32:03 IST 2021
```

Comment:
`java.util.Properties` has this nasty 'feature' to unconditionally write this header. I would very much love to not have it.